src/strategies/dappnode-dao/examples.json

```
@@ -0,0 +1,26 @@
+[
+  {
+    "name": "DAppNodeDAO staked in Unipool rewards contracts multichain",
+    "strategy": {
+      "name": "dappnode-dao",
+      "params": {
+        "symbol": "NODE",
+        "decimals": 18,
+        "tokenAddress": "0xbC396689893D065F41bc2C6EcbeE5e0085233447",
+        "unipoolAddress": "0xb9840A4A8a671f79DE3Df3b812fEEb38047CE552",
+        "bpoolAddress": "0xf54025af2dc86809be1153c1f20d77adb7e8ecf4",
+        "unipool1": "0xbC396689893D065F41bc2C6EcbeE5e0085233447",
+        "unipool2": "0xbC396689893D065F41bc2C6EcbeE5e0085233447",
+        "unipoolSameToken": "0xbC396689893D065F41bc2C6EcbeE5e0085233447",
+        "unipoolSameTokenXdai": "0xbC396689893D065F41bc2C6EcbeE5e0085233447"
+      }
+    },
+    "network": "1",
+    "addresses": [
+      "0x09212c58107c8da21b0f67a63e2144bb68bee4eb",
+      "0x8506d946CC63d1F1F3a303d68B0Da64597Cd64F3",
+      "0xc5df4d5ed23f645687a867d8f83a41836fcf8811"
+    ],
+    "snapshot": "11635000"
```

This should be a number
```suggestion
    "snapshot": 11635000
```

---

Removed un-related second strategy from this PR